stdlib/3/_ast.pyi

```diff
@@ -3,6 +3,8 @@ import typing
 from typing import Any, Optional, ClassVar
 
 PyCF_ONLY_AST: int
+PyCF_TYPE_COMMENTS: int
+PyCF_ALLOW_TOP_LEVEL_AWAIT: int
```

Comment:
These are only 3.8+ right? I'm not insisting on another `if`, just curious if you have a specific policy about when to make things version-specific (e.g. you do it for the MADV_* flags in mmap.pyi.

Reply:
Good catch, added a version check.